examples/bri-3/src/bri/workgroup/workgroups/agents/workgroupStorage.agent.ts

```diff
@@ -19,7 +19,7 @@ export class WorkgroupStorageAgent extends PrismaService {
         administrators: true,
         participants: true,
         workflows: {
-          include: { worksteps: true },
+          include: { worksteps: true, bpiAccount: true },
```

Comment:
`bpiAccountId`?

Reply:
slightly confused by the difference of bpiAccount which should be an object and bpiAccountId which should be a string

Reply:
Just a convention picked up from prisma.schema. I do not think we need both of these in the domain object (object reference should be enough) but i cant change that right now across the whole codebase. Let's just be consistent in how we do it and i ll open a separate issue to tackle it.